app/models/order.rb

```diff
@@ -0,0 +1,5 @@
+class Order < ApplicationRecord
+  belongs_to :user
+  has_many :order_lists, dependent: :destroy
+  has_many :products, through: :order_lists
```

Comment:
thêm dependent: :destroy vào tất cả has_many em nhé

Reply:
Cái này không thể được anh ơi. Ở đây, vì một số has_many em có sử dụng through nữa, tức là diễn tả quan hệ many-to-many thông qua bảng trung gian. Chính vì vậy như ví dụ trên thì dù order có bị xoá thì products cũng k bị xoá theo ạ. Chỉ có orderlist(là trung gian giữa order với product là có thể xoá theo).